.github/stale.yml

```diff
@@ -28,20 +28,18 @@ staleLabel: stale
 
 # Comment to post when marking as stale. Set to `false` to disable
 markComment: >
-  This issue has been automatically marked as stale because it has no recent
-  activity in the last 60 days. It will be closed in 14 days if no further
-  activity occurs. If this issue is still relevant please leave a comment to
-  let us know and the stale label will be automatically removed. Thank you for
-  your contributions.
```

Comment:
This was our wording, we preferred as developers to have reasoning and timelines for when these actions would occur.

I'm happy to move the thank you to the beginning of the statement.